src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadoctype/InputJavadocTypeWithBlockComment.java

```diff
@@ -0,0 +1,28 @@
+/*
+JavadocType
+scope = (default)private
+excludeScope = (default)null
+authorFormat = (default)null
+versionFormat = (default)null
+allowMissingParamTags = (default)false
+allowUnknownTags = (default)false
+allowedAnnotations = (default)Generated
+tokens = (default)INTERFACE_DEF, CLASS_DEF, ENUM_DEF, ANNOTATION_DEF, RECORD_DEF
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.javadoc.javadoctype;
+
+
+/**
+ *  InputJavadocTypeUnusedParamInJavadocForClass.
+ *
+ *  @param <T> This is type parameter. // violation 'Unused @param tag for '<T>'.'
```

Comment:
Please put violation comment above javadoc, to let javadoc look natural.
You can use `// violation below 4 lines ....`